source/_posts/2025-10-22-voice-chapter-11.markdown

```diff
@@ -0,0 +1,129 @@
+
+[Assist](/voice_control/), our built-in voice assistant, can do a lot of cool things without the need for AI! This includes [a ton of voice commands in dozens of languages](/voice_control/builtin_sentences/) for:
+
+* Turning lights and other devices on/off  
+* Opening/closing and locking/unlocking doors, windows, shades, etc  
+* Adjusting the brightness and color of lights  
+* Running scripts and activating scenes  
+* Controlling media players and adjusting their volume  
+* Playing music on supported media players via [Music Assistant](/integrations/music_assistant/)  
+* Starting/stopping/pausing multiple timers, optionally with names  
+* Adding/completing items on to-do lists  
+* Delaying a command for later (“turn off lights in 5 minutes”)...  
+* …and more!
+
+Want to include your own voice commands? You can quickly add [custom sentences](/voice_control/custom_sentences/) to an automation, allowing you to take any action and tailor the response.
+
+The easiest way to get started is with [Home Assistant Voice Preview Edition](/voice-pe/), our small and easy-to-start with Voice Assistant hardware. This, combined with a [Home Assistant Cloud subscription](/cloud/), allows any Home Assistant system to quickly handle voice commands, as our privacy-focused cloud processes the speech-to-text (turning your voice into text for Home Assistant) and text-to-speech (turning Home Assistant’s response back into voice). This is all without the use of LLMs, and supports the development of Home Assistant 😎.
+
+For users wanting to keep all voice processing local, we offer add-ons for both speech-to-text and text-to-speech:
+
+* [Whisper](https://github.com/home-assistant/addons/blob/master/whisper/DOCS.md) is a powerful speech-to-text system that comes in [different sizes with varying hardware requirements](https://github.com/openai/whisper#available-models-and-languages)  
+* [Speech-to-Phrase](/blog/2025/02/13/voice-chapter-9-speech-to-phrase/) is our speech-to-text system that trades flexibility for speed  
+* [Piper](https://github.com/home-assistant/addons/blob/master/piper/DOCS.md) is our fast neural text-to-speech system with [broad language support](https://rhasspy.github.io/piper-samples/)
+
+All of this together shows just how much can be done without needing to include AI, even though it can do [some pretty amazing things](https://youtu.be/mLtFUG4YG1A?si=YRTRtSdv5ySU_JBJ). And we’re continuing to close the gap with the features highlighted in this blog post, including multilingual assistants, improved sentence matching, and the ability to ask questions from automations.
```

Comment:
```suggestion
All of this together shows just how much can be done without needing to include AI, even though it can do [some pretty amazing things](https://youtu.be/mLtFUG4YG1A). And we’re continuing to close the gap with the features highlighted in this blog post, including multilingual assistants, improved sentence matching, and the ability to ask questions from automations.
```